packages/flutter/lib/src/rendering/box.dart

```diff
@@ -2147,10 +2277,15 @@ abstract class RenderBox extends RenderObject {
     super.markNeedsLayout();
   }
 
+  /// {@macro flutter.rendering.RenderObject.performResize}
+  ///
+  /// By default this method calls [getDryLayout] with the current
+  /// [constraints]. Instead of overriding this method, consider overriding
+  /// [computeDryLayout] (which is used by [getDryLayout] behind the scenes).
```

Comment:
"(which is used by [getDryLayout] behind the scenes)" sounds mysterious, maybe "(the backend implementation of getDryLayout)" or something?